app/api/health_api.py

```
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from app.db.mongo import db
+
+router = APIRouter()
+
+@router.get("/health")
+def health_check():
+    try:
+        db.command("ping")
+        return {"status": "MongoDB connected"}
+    except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)